client/cypress/support/commands.js

```diff
@@ -30,3 +30,16 @@ Cypress.Commands.add('fillInputs', (elements) => {
     cy.getByTestId(testId).clear().type(value);
   });
 });
+
+Cypress.Commands.add(
+  'percyMultipleSnapshots',
+  (name, { widths = [1280, 375], minHeight = 1024, takeSnapshotDelay = null } = {}) => {
+    each(widths, (width) => {
+      cy.viewport(width, minHeight);
+      if (takeSnapshotDelay > 0) {
+        cy.wait(takeSnapshotDelay); // eslint-disable-line cypress/no-unnecessary-waiting
+      }
+      cy.percySnapshot(name, { widths: [width], minHeight });
```

Comment:
I'm curious to see if Percy will recognize this as the same image with different widths

Reply:
he-he, I'm curious as well :slightly_smiling_face: we will know it in few minutes :slightly_smiling_face:

Reply:
huh, no, it does recognize only first screenshot. crap :thinking: